docker-compose.yml

```diff
@@ -45,6 +45,8 @@ services:
   rems-smart-on-fhir:
     image: codexrems/rems-smart-on-fhir:REMSvCurrent
     container_name: rems_prod_rems-smart-on-fhir
+    env_file:
+      - ../rems-smart-on-fhir/.env
```

Comment:
env_file is fine for the docker-compose.dev file but for the regular docker-compose file this should use the environment field to manually set any individual fields that would be different than the defaults for running locally on one machine. Reason for this is because this file is used for the production install by sponsors which only has them clone the REMS repo (as they don't need the rest). The .env file in rems-smart-on-fhir should also already be in the published container anyways so those defaults should be read in by that app anyways. Also another note to make is that this .env file will change to an env.json file in a future ticket